scripts/ddconfig.sh

```diff
@@ -18,6 +18,22 @@ function DisableAndExit() {
     exit 0
 }
 
+function PrintEndpointAddress() {
```

Comment:
I'm not sure I follow, I called it PrintEndpointAddress because thats what its doing, and that seemed to be a conventional way to return values.

I don't think jq is a standard tool, in this case the tool will be used by other people so we don't really control the environment.

You're right about attempting to use algocfg and then falling back, but the original script would have failed in this case anyway. And probably once we get the next protocol upgrade out we can remove this logic completely.